aat.tfvars

```diff
@@ -0,0 +1 @@
+allowed_jurisdictions = "'ia', 'IA', 'sscs', 'SSCS', 'wa', 'WA', 'civil', 'CIVIL', 'publiclaw', 'PUBLICLAW', 'privatelaw', 'PRIVATELAW', 'employment','EMPLOYMENT'"
```

Comment:
I think the intention was to keep AAT similar to prod rather than have everyone there?
Maybe just those with an AtO and ourselves to start with.  Is that going to cause a problem elsewhere?

Reply:
will remove the other values